### ISSUE TITLE
Log notes or recordings on a brew timeline

### ISSUE THREAD

**Issue by meirion** [thorrak/fermentrack#312]
Suggestion...

When I'm brewing, I might make measurements (gravity) or observations at certain days during the fermentation. Perhaps there was a power cut or the fridge door was left open which affected the temperature, and I wanted to note down when that was, or I want to record the point at which additional hops went it, or what the manual gravity measurement was on a certain day (spindel not working yet)

It would be good to add day/time logged notes as the brew progressed. These could be indicated also on the graph, as points that would reveal a title when clicked.

Basically trying to get more done with Fermentrack and rely less on manual methods.

Ideally I'd be taking photos as well and be able to upload these.... and then it would become a cloud based platform, I could access it remotely, and I could share and publish brews ... and I'l stop dreaming now!
